libexec/tgenv-upgrade

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+[ -n "${TGENV_DEBUG}" ] && set -x
+source "${TGENV_ROOT}/libexec/helpers"
+
+echo "Upgrading tgenv to the latest version..."
+
+if [ -d "$HOME/.tgenv" ]; then
+  cd "$HOME/.tgenv"
+  git checkout main
+  git pull origin main
+else
+  echo "tgenv is not installed. Please install it before running this script."
```

Comment:
Consider whether this still makes sense in light of the other comments

Reply:
I've "improve" the comment 😄 .